dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/AccessTokenMapper.java

```diff
@@ -61,4 +61,12 @@ IPage<AccessToken> selectAccessTokenPage(Page page,
      * @return delete result
      */
     int deleteAccessTokenByUserId(@Param("userId") int userId);
+
+    /**
+     * list authorized Projects
+     * @param userId
+     * @param accessTokensIds
+     * @return access token for specified user
+     */
+    List<AccessToken> listAuthorizedAccessToken(@Param("userId") int userId, @Param("accessTokensIds")List<Integer> accessTokensIds);
```

Comment:
The length of the mapped SQL judgment set should be the same as the method parameter type, SQL uses size() or the method parameter is changed to an array.